nixos/modules/services/misc/sourcehut/sourcehut.xml

```diff
@@ -0,0 +1,114 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
```

Comment:
This should be done in markdown.

Reply:
backlog this to another PR. this has already gotten too bloated